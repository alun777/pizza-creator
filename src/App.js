import React from 'react';
import './index.css';
import PizzaCreatorAppForm from './components/PizzaCreatorAppForm';


function App() {
  return (
    <PizzaCreatorAppForm></PizzaCreatorAppForm>
  );
}

export default App;
