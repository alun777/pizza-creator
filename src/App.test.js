import App from './App'
import React from 'react';
import { shallow } from 'enzyme';

it('renders without crashing', ()=>{
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
