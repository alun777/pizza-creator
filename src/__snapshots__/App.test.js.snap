// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(PizzaCreatorAppForm)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
  </BrowserRouter>
</Provider>
`;
